pkg/asset/apiservice.go

```diff
@@ -29,9 +29,8 @@ func (a *apiservice) New() *apiregistrationv1.APIService {
 		},
 		ObjectMeta: metav1.ObjectMeta{
 			Name: a.Name(),
-			Labels: map[string]string{
-				a.values.OwnerLabelKey:   a.values.OwnerLabelValue,
```

Comment:
I think we might still want the owner label?  but good call on removing the other one.

Reply:
Sure, what do these labels do?

Reply:
- the owner label just tells things which operator owns and is reconciling the object, so if someone runs across an object somewhere and wants to know where it came from (and maybe why it keeps coming back if they delete it), it's easy to figure out instead of having to reverse engineer from managed fields or read through audit logs or something. It's a nicety, not a hard requirement of anything, but it's our operator being a "good citizen" of the cluster.  
- the `kube-aggregator.kubernetes.io/automanaged` label I believe should only be used for things the API server creates automatically, and it will do things like clean those up if it doesn't like the way they are, so my understanding is we don't want it since we are creating this object ourselves. See: https://kubernetes.io/docs/reference/labels-annotations-taints/#kube-aggregator-kubernetesio-automanaged